documentation/src/main/asciidoc/user_guide/chapter-78-Scripting.adoc

```diff
@@ -11,7 +11,14 @@ Alternatively the script engine can be selected using script metadata (see below
 
 ==== Script metadata
 Script metadata is additional information about the script that the user can provide to the server to affect how a script is executed. 
-Script metadata is contained in a specially-formatted comment on the first line of the script.
+It is contained in a specially-formatted comment on the first lines of the script. Properties are specified as +key=value+ pairs, separated by commas.
+You can use several different comment styles: The '//', ';;', '#' depending on the scripting language you use. You can split metadata over multiple lines if necessary, and you can use single (') or double (") quotes to delimit your values.
```

Comment:
It looks too big for a single paragraph, I think you need to add some empty lines. Also, I'd replace "The '//', ';;', '#'" with "`//`, `;;`, or `#`,"